app/templates/pdf/ticket_attendee.html

```diff
@@ -1,96 +1,212 @@
+                        {% elif order.event.location_name %}
+                        <span>{{_('Event taking place at
+                            %(location)s',location="")}}{{order.event.location_name}}</span>
+                        {% else %}
+                        <span>{{_('Location or online event details to be announced')}}</span>
+                        {% endif %}
+                    </span>
+                    <br>
+                    <span>
+                        {% if order.event.starts_at %}
+                        <span class="time">{{_('From: %(start_date)s',start_date="")}}{{order.event.starts_at |
+                            datetime(order.event.timezone)}}</span>
+                        {% endif %}
+                        <br>
+                        {% if order.event.ends_at %}
+                        <span class="time">{{_('To: %(end_date)s',end_date="")}}{{order.event.ends_at |
+                            datetime(order.event.timezone)}}</span>
+                        {% endif %}
+                    </span>
+                    <span>
+                        <table id="order_details">
+                            <tr>
+                                <td>
+                                    <span class="custom_label">{{_('Order Information')}}</span>
+                                    <p class="data">{{_('Order
+                                        #%(invoice_number)s',invoice_number="")}}{{order.get_invoice_number()}}</p>
+                                </td>
+                                <td>
+                                    <span class="custom_label">{{_('Name')}}</span>
+                                    <p class="data">{{holder.name}}</p>
+                                </td>
+                            </tr>
+                        </table>
+                    </span>
+                </td>
+                <td align="center">
+                    <!-- <img src="{{order.event.original_image_url}}" alt="" id="event-logo"><br><br><br> -->
+                    <img src="{{order.event.original_image_url}}" alt=""
+                        id="event-logo"><br><br><br>
 
-        <div id="qrcode">
-            <img src="data:image/png;base64,{{ holder.qr_code }}" />
-        </div>
-        <strong id="number_content" style="color: #000; text-align: center">#{{ order.get_invoice_number() }}/{{ holder.id }}</strong>
+                    <div id="qrcode">
+                        <img src="data:image/png;base64,{{ holder.qr_code }}" />
+                        <br>
+                        <span>#{{order.get_invoice_number()}}</span>
+                    </div>
+                </td>
+            </tr>
+        </table>
+    </div>
+    {% if order.event.after_order_message %}
+    <div class="main_content" id="footer">
+        <span id="event_message">{{ order.event.after_order_message | replace("\n","<br>") | safe }}</span>
+    </div>
+    {% endif %}
+</body>
+
```

Comment:
Where is the `Type` & `holder.email` ??

Reply:
Shall I change Guest to Holder Ticket name

Reply:
No, just add `Type` as before.